### DISCUSSION TITLE
Setting up version 11 on a MAC

### DISCUSSION THREAD

**Discussion by zt1947** [Open-Systems-Pharmacology/Forum#1250] [Questions & Problems]
I have downloaded OSPSuite-Full.11.0.139.exe to my MAC Tablet. How doI open it to start running PK-Sim & MOBI

**Comment by Yuri05** [comment]
MAC is not supported. However you still could use the OSP Suite on MAC using e.g. different virtualization approaches
s. recent discussion here: https://github.com/Open-Systems-Pharmacology/Forum/discussions/1209
